lib/propcheck.ex

```diff
@@ -383,6 +383,8 @@ defmodule PropCheck do
 
     defp forall_impl(var, rawtype, opts, prop) do
       quote do
+        property_opts = Process.get(:property_opts, [])
```

Comment:
That's the first that Joe Armstrong will turn around in his grace. Yuck. Not that I have a better though.

Reply:
Yes, I don't like that approach very much either. I spent a lot of time thinking about how to pass the options from a surrounding property into `forall`, but I could not make it work. It is non-obvious to detect within a macro that it is called within another macro and expose the environment of the surrounding macro. At least, the approach here is dead simple and easy to understand.